test/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I've created a webpack config for dev env and it includes the plugins `html-webpack-template` and `html-webpack-plugin`. So we don't need a html file anymore to test the project locally !!